bastion/examples/restart_strategy.rs

```diff
@@ -0,0 +1,31 @@
+use std::time::Duration;
+
+use bastion::prelude::*;
+
```

Comment:
We mostly write a prelude intro to our examples to explain where an example can be used for and what's the intent. I would like to have that. You might want to take a gaze to this one:
e.g. https://github.com/bastion-rs/bastion/blob/master/bastion/examples/parallel_computation.rs#L4

Reply:
Done.